src/http.cc

```diff
@@ -117,6 +119,14 @@ int HttpPost(const std::string &url, const std::string &data,
   // Network.
   curl_easy_setopt(curl, CURLOPT_URL, url.c_str());
 
+  // Certs.
+  if (ca_certs_dir.empty()) {
+    if (!ca_bundle_file.empty())
+      curl_easy_setopt(curl, CURLOPT_CAINFO, ca_bundle_file.c_str());
```

Comment:
I think this might be more readable if instead you wrote:

```
if (ca_certs_dir.empty()) {
  if (ca_bundle_file.empty()) {
    // Use default CA location.
  } else {
    curl_easy_setopt(...);
  }
} else {
  curl_easy_setopt(...);
}